opengemini-client-jdk/src/main/java/io/opengemini/client/okhttp/OpenGeminiJdkClientFactory.java

```diff
@@ -1,4 +1,4 @@
-package io.opengemini.client.jdk;
```

Comment:
Will this rename cause the test cases for the JDK components to be missing? It seems that there is another PR #49 that is extracting common test modules, and these two PRs may conflict.